roles/rhel_subscribe/tasks/enterprise.yml

```diff
@@ -2,18 +2,11 @@
 - name: Disable all repositories
   command: subscription-manager repos --disable="*"
 
-- set_fact:
-    default_ose_version: '3.6'
-  when: deployment_type == 'openshift-enterprise'
-
-- set_fact:
-    ose_version: "{{ lookup('env', 'ose_version') | default(default_ose_version, True) }}"
-
 - fail:
```

Comment:
This check can go away as openshift-enterprise is for all of our currently supported releases.

Also, we need to sync up on using openshift_release.  default_ose_version needs to go away.